Readme.md

```diff
@@ -132,15 +139,20 @@ see where it's stored) if you don't plan to reinstall Dolphin.
 * -V, --video_backend=<str> Specify a video backend
 * -A, --audio_emulation=<str> Low level (LLE) or high level (HLE) audio
 
-Available DSP emulation engines are HLE (High Level Emulation) and
-LLE (Low Level Emulation). HLE is fast but often less accurate while LLE is
-slow but close to perfect. Note that LLE has two submodes (Interpreter and
-Recompiler), which cannot be selected from the command line.
+When the `-d` argument is used, debugger features can be viewed from the "View" menu.
+Execution breakpoints are available in the "Release" build of Dolphin.
+Memory read/write breakpoints are available in DEBUGFAST builds of Dolphin.
```

Comment:
That's incorrect AFAIK. You can use MBPs in release builds (on Linux, at least).